devicetypes/qubino/qubino-dimmer.src/qubino-dimmer.groovy

```diff
@@ -394,6 +389,14 @@ private dimmerEvents(physicalgraph.zwave.Command cmd, ep = null) {
 	return result
 }
 
+Integer adjustValueToRange(value){
+	if(value == 0){
+		return 0
+	}
+	def minDimmingLvlPref = settings.minimumDimmingValue ?: parameterMap.find({it.key == 'minimumDimmingValue'}).defaultValue
+	return Math.max(value, minDimmingLvlPref)
```

Comment:
won't you get a "network error" for the same reason if you try to set the level lower than the minimum dimming value? i.e. if I set level to 15 but my minimum dimming level is 20, the value I get returned is 20. How is that any better?

Reply:
There is no network errors in application. When set level is lower than minimum dimming level, then the level set by the device is minimum dimming level. We do not get network error, we change the level value.